code/datums/station_traits/innate_traits.dm

```diff
@@ -0,0 +1,107 @@
+//This file is for innate station traits, which are applied from the map!
+
+#define TREK_VARIATION_TOS "tos"
+#define TREK_VARIATION_TNG "next"
+#define TREK_VARIATION_VOY "voy"
+#define TREK_VARIATION_ENT "ent"
+
+/datum/station_trait/trek
+	name = "Exploratory Uniforms"
+	trait_type = STATION_TRAIT_NORANDOM
```

Comment:
@MrStonedOne FYI - This trait is automatically applied to this map by default via the JSON, which does bypass the `#GREY_ASSISTANTS` configuration. I don't think it matters too much at the end of the day, but given that you both introduced it and are the final say on configurationy stuff, do you think it matters?